src/OrchardCore/OrchardCore/Shell/Distributed/DistributedShellHostedService.cs

```diff
@@ -185,7 +185,7 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
                             }
 
                             // Try to retrieve the reload identifier of this tenant from the distributed cache.
-                            var reloadId = await distributedCache.GetStringAsync(ReloadIdKey(settings.Name));
+                            var reloadId = await distributedCache.GetStringAsync(ReloadIdKey(settings.Name), stoppingToken);
```

Comment:
So, I prefer to not change this file.

I will add the stoppingToken in my PR #11890.

I already saw the analyzer suggesting it but was too lazy ;)

Reply:
Got it! I’ll revert that change.